src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -54,61 +56,69 @@ def test_unfinished_run(caplog, testfiles):
 
 
 def test_no_atomic_section(caplog, testfiles):
-    """Test if we give a warning if there is no ATOMIC section"""
+    """Test if raise an exception if there is no ATOMIC section"""
     filename = testfiles["no_atomic_section"][0]
-    with caplog.at_level(logging.WARNING):
-        _ = extract(str(filename), T=298.0)
-    assert "No ATOMIC section found" in caplog.text
+    with pytest.raises(ValueError, match='no "ATOMIC" section found'):
+        with caplog.at_level(logging.ERROR):
+            _ = extract(str(filename), T=298.0)
+    assert 'no "ATOMIC" section found' in caplog.text
```

Comment:
Match capitalization of sentences (as suggested in my comments):
```suggestion
    assert 'No "ATOMIC" section found' in caplog.text
```